crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -313,19 +321,10 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for TrustpayPaymentsRequest {
                 item.request.get_return_url()?,
             )),
             api::PaymentMethodData::BankRedirect(ref bank_redirection_data) => {
-                Ok(get_bank_redirection_request_data(
-                    item,
-                    bank_redirection_data,
-                    amount,
-                    item.request.get_return_url()?,
-                    auth,
-                ))
+                get_bank_redirection_request_data(item, bank_redirection_data, amount, auth)
             }
-            _ => Err(errors::ConnectorError::NotImplemented(format!(
-                "Current Payment Method - {:?}",
-                item.request.payment_method_data
-            ))),
-        }?)
+            _ => Err(errors::ConnectorError::NotImplemented("Payment methods".to_string()).into()),
```

Comment:
We should log the payment method for better debugging purpose

Reply:
as discussed we need not to take this change